pyproject.toml

```diff
@@ -36,7 +36,8 @@ dependencies = [
    "jsonschema",
    "requests",
    "tqdm",
-   "datasets",
+   "typing_extensions",
+   "torch",
```

Comment:
We rewrote the code so we don't have to force users who use models that don't require `torch` to run outlines without installing `torch`, which is a big dependency.